discord/client.py

```diff
@@ -573,7 +573,7 @@ def start(self, email, password):
         yield from self.login(email, password)
         yield from self.connect()
 
-    def run(self, email, password):
+    def run(self, email, password, token):
```

Comment:
I added token for those who did not update their account yet as the "Bot" account type does not have a UI and if they fuck up they would have to make a new account.